builds/fabric8-che/pom.xml

```diff
@@ -18,7 +18,6 @@
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
         <version>5.18.0-SNAPSHOT</version>
```

Comment:
Afaik we wanted to use tag version of upstream not snapshot or this is separate thing ?